go/cmd/vtctldclient/command/vreplication/common/mirrortraffic.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"vitess.io/vitess/go/cmd/vtctldclient/cli"
+
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
+)
+
+func GetMirrorTrafficCommand(opts *SubCommandsOpts) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "mirrortraffic",
+		Short:                 fmt.Sprintf("Mirror traffic for a %s VReplication workflow.", opts.SubCommand),
+		Example:               fmt.Sprintf(`vtctldclient --server localhost:15999 %s --workflow %s --target-keyspace customer mirrortraffic --percent 50.0`, opts.SubCommand, opts.Workflow),
```

Comment:
```suggestion
		Example:               fmt.Sprintf(`vtctldclient --server localhost:15999 %s --workflow %s --target-keyspace customer mirrortraffic --percent 5.0`, opts.SubCommand, opts.Workflow),
```

Reply:
I always assume that people will copy-paste examples and try to use a value that we'd actually recommend. I'm suggesting 5% here, but maybe you think 10% might be better, which is also fine. It just seems unlikely that people would start with 50%, but if you think that is likely, I don't have a problem with it.

Reply:
Once a MirrorTraffic command has been issued, is there a way to increase the percentage?
What happens if someone has already issued a MirrorTraffic command for a from->to keyspace pair, and issue the same command (or similar with a different percentage value)?
The answers to these questions should be validated with unit tests.

Reply:
> Once a MirrorTraffic command has been issued, is there a way to increase the percentage?

yep! just by re-issuing the same command with a different %.

> What happens if someone has already issued a MirrorTraffic command for a from->to keyspace pair, and issue the same command (or similar with a different percentage value)?

the % of mirrored traffic will be increased to the new value.

> The answers to these questions should be validated with unit tests.

sounds good will do.

Reply:
done in [71b9a00](https://github.com/vitessio/vitess/pull/15945/commits/71b9a008952a81fa325249efdf5658a92991c92d)